### DISCUSSION TITLE
I'm trying to let the maintainers know that they forgot to include "unstable_cacheTag" in their new release. Didn't they ask for community feedback?

### DISCUSSION THREAD

**Discussion by techmangreat** [vercel/next.js#71942] [App Router]
The requirements for opening an issue are ridiculous. Mine was closed automatically. I won't create a github repo just for that. I'm trying to help and give quick feedback on something that they were happy to announce and ask for feedback. Is that how the maintainers want to treat the community and the pioneers on the canary versions?

#71845

**Comment by icyJoseph** [comment]
Did you follow these steps?

> To play with it, you must be on the canary version of Next.js:

```console
npx create-next-app@canary
```

And:

```ts
import type { NextConfig } from 'next';
 
const nextConfig: NextConfig = {
  experimental: {
    dynamicIO: true,
  }
};
 
export default nextConfig;
```

People are constantly spamming the Issues on large repositories, not just Next.js. Anything from scams, spam and etc, so having the reproduction repository as a requisite, is almost a necessary step to prevent bad actors. I've seen the Next.js issues having pages upon pages of just spam, only in recent weeks. People do find ways around it.

The discussions do exist for what you wanted, to give feedback!
